data/translations/en/state-data.json

```diff
@@ -749,27 +834,31 @@
     "more_info_link": "https://www.scvotes.gov/south-carolina-voter-registration-information",
     "more_info_link_translated": "true",
     "more_info_link_language_selection": "",
-    "ip_deadline": "30 days before Election Day",
-    "online_deadline": "30 days before Election Day",
-    "mail_received_deadline": "",
-    "mail_postmarked_deadline": "Must be postmarked 30 days before Election Day",
+    "online_deadline": "Sunday, October 09, 2022",
+    "ip_deadline": "Sunday, October 09, 2022",
+    "mail_received_deadline": "Must be received by Tuesday, October 11, 2022",
```

Comment:
Has both received and postmarked deadline. Which one should we use?

Reply:
Updated to use only received

Reply:
confirmed